vector/src/main/java/im/vector/util/ThemeUtils.kt

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright 2017 OpenMarket Ltd
```

Comment:
You should set here the 2018 copyright here

Reply:
done